tests/test_snql_api.py

```diff
@@ -0,0 +1,74 @@
+import pytz
+import uuid
+from datetime import datetime, timedelta
+from functools import partial
+
+import simplejson as json
+
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_writable_storage
+from tests.base import BaseApiTest
+from tests.fixtures import get_raw_event, get_raw_transaction
+from tests.helpers import write_unprocessed_events
+
+
+class TestSnQLApi(BaseApiTest):
+    def setup_method(self, test_method):
+        super().setup_method(test_method)
+        self.app.post = partial(self.app.post, headers={"referer": "test"})
+        self.trace_id = uuid.UUID("7400045b-25c4-43b8-8591-4600aa83ad04")
+        self.event = get_raw_event()
+        self.project_id = self.event["project_id"]
+        self.skew = timedelta(minutes=180)
+        self.base_time = datetime.utcnow().replace(
+            minute=0, second=0, microsecond=0, tzinfo=pytz.utc
+        ) - timedelta(minutes=180)
+        write_unprocessed_events(get_writable_storage(StorageKey.EVENTS), [self.event])
```

Comment:
This should eventually have actual data for the JOIN, but for now I just wanted to make sure the queries were valid so William can start playing with it. I'll expand these tests as we go.